packages/components/src/overlays/Popover/src/Popover.module.css

```diff
@@ -0,0 +1,96 @@
+:where(.hop-Popover *) {
+    font-family: var(--font-family);
+    font-size: var(--font-size);
+    font-weight: var(--font-weight);
+    line-height: var(--line-height);
+    color: var(--color);
+
+    outline: none;
```

Comment:
does this need to be set on everything except for .hop-Popover? I'd think it should be set on .hop-Popover and maybe the dialog?

Reply:
I don't quite understand what you mean. Are you talking about adding a focus state to the Popover or Dialog?

Reply:
no I'm just asking why outline: none; is set on each child?

Reply:
Child elements have their own state for displaying the focus, and are not affected by the `outline`.